scripts/sanitycheck

```diff
@@ -2693,6 +2693,7 @@ class TestSuite:
 
         discards = {}
         platform_filter = kwargs.get('platform')
+        exclude_platform = kwargs.get('exclude_platform')
```

Comment:
No need to use `get()` if you know the key exists.

`if plat.name in exclude_platform` below would crash if the key doesn't exist.

Reply:
thanks, will remove all of them in another patch.